app/models/partner.rb

```diff
@@ -288,6 +288,10 @@ def twitter_url
     "https://twitter.com/#{twitter_handle}" if twitter_handle.present?
   end
 
+  def instagram_url
+    "https://instagram.com/#{instagram_handle}" if instagram_handle.present?
```

Comment:
Couldn't immediately see where the `instagram_handle` is coming from - but appears it was already part of the model. Was it previously not being rendered as a link?